DISTRIBUTING.md

```diff
@@ -138,8 +138,8 @@ called `USE_BLAS64` is available as part of the Julia build. When doing
 `make USE_BLAS64=0`, Julia will call BLAS and LAPACK assuming a 32-bit
 API, where all integers are 32-bit wide, even on a 64-bit architecture.
 
-Other libraries that Julia uses, such as ARPACK and SuiteSparse also
-use BLAS and LAPACK internally. The APIs need to be consistent across
+Other libraries that Julia uses, such as SuiteSparse (and Arpack.jl that was previously an
```

Comment:
Probably not worth mentioning ARPACK at all anymore since it isn't part of Julia and is thus no longer relevant.